File: test/Chess.c
#include <turtle.h>

int main(void)
{
    for (int i = 0; i < 8; i++)
    {
        up();
        goto(-200, -200 + 50 * i);
        down();
	
        for (int j = 0; j < 8; j++)
        {
            if (!((i + j) % 2))
                color("brown");
            else
                color("yellow");

            begin_fill();
            for (int k = 0; k < 4; k++)
            {
                forward(50);
                left(90);
            }
            end_fill();
            up();
            forward(50);
            down();
        }
    }

    show();
    return 0;
}

File: samples/python/Chess.py
from turtle import *

speed(0)
	
for i in range(8):
	up()
	goto(-200, -200 + 50 * i)
	down()
	for j in range(8):
		if not (i + j) % 2:
			color('black')
		else:
			color('brown')

		# begin_fill()
		for k in range(4):
			forward(50)
			left(90)

		# end_fill()
		forward(50)

hideturtle()
mainloop()